src/pages/App.tsx

```diff
@@ -63,6 +63,10 @@ const AMPLITUDE_PROXY_URL = process.env.REACT_APP_AMPLITUDE_PROXY_URL
 const STATSIG_DUMMY_KEY = 'client-0000000000000000000000000000000000000000000'
 const STATSIG_PROXY_URL = process.env.REACT_APP_STATSIG_PROXY_URL
 const COMMIT_HASH = process.env.REACT_APP_GIT_COMMIT_HASH
+
+// Dump some metadata into the window to allow client verification.
+window.GIT_COMMIT_HASH = COMMIT_HASH
```

Comment:
nit: not that have any particular preference, but just wanted to point out that we seem to be touching global namespace / interacting with top-level code in a few places already, particularly in `index.tsx` (where I configure Sentry or we set `window.ethereum.autoRefreshOnNetworkChange`) or `polyfills.ts`.  I think it would make sense to move it one level up, to be side by side `react-app-env.d.ts`.

Reply:
I don't disagree - I was going to move around the 3P initializations in a follow-up, and I could include this as well.